src/commands/ai/extract.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const BoxCommand = require('../../box-command');
+const {Flags} = require('@oclif/core');
+const utils = require('../../util');
+
+class AiExtractCommand extends BoxCommand {
+	async run() {
+		const {flags} = await this.parse(AiExtractCommand);
+		let options = {};
+
+		if (flags.prompt) {
+			options.prompt = flags.prompt;
+		}
+		if (flags.items) {
+			options.items = flags.items;
+		}
+		if (flags['ai-agent']) {
+			options.aiAgent = flags['ai-agent'];
+		}
+
+		let answer = await this.tsClient.ai.createAiExtract(options);
+
+		delete answer.rawData;
+		await this.output(answer);
+	}
+}
+
+AiExtractCommand.description =
+	'Sends an AI request to supported LLMs and extracts metadata in the form of key value pairs';
+AiExtractCommand.examples = [
+	'box ai:extract --items=id=12345,type=file --prompt "firstName, lastName, location, yearOfBirth, company"',
+	'box ai:extract --prompt "firstName, lastName, location, yearOfBirth, company" --items "id=12345,type=file" --ai-agent \'{"type":"ai_agent_extract","basicText":{"llmEndpointParams":{"type":"openai_params","frequencyPenalty": 1.5,"presencePenalty": 1.5,"stop": "<|im_end|>","temperature": 0,"topP": 1},"model": "azure__openai__gpt_4o_mini","numTokensForCompletion": 8400,"promptTemplate": "It is, consider these travel options and answer the.","systemMessage": "You are a helpful travel assistant specialized in budget travel"},"longText":{"embeddings":{ "model": "azure__openai__text_embedding_ada_002","strategy":{"id": "basic","numTokensPerChunk": 64}},"llmEndpointParams":{"type":"openai_params","frequencyPenalty": 1.5,"presencePenalty": 1.5,"stop": "<|im_end|>","temperature": 0,"topP": 1},"model":"azure__openai__gpt_4o_mini","numTokensForCompletion":8400,"promptTemplate":"It is , consider these travel options and answer the.","systemMessage":"You are a helpful travel assistant specialized in budget travel"}}\'',
+];
+AiExtractCommand._endpoint = 'post_ai_extract';
+
+// Flags definition
+AiExtractCommand.flags = {
+	...BoxCommand.flags,
+	prompt: Flags.string({
+		required: true,
+		description: 'The prompt for the AI request',
+	}),
+	items: Flags.string({
+		required: true,
+		description: 'The items for the AI request',
```

Comment:
Reuse description from Box API, but short version (should the first sentence, for example)